src/Badge/Badge.js

```diff
@@ -0,0 +1,115 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from '../utils/customPropTypes';
+
+const radius = 12;
+const radius2x = Math.floor(2 * radius);
```

Comment:
@stunaz It rounds down to the nearest integer, which doesn't make much sense for integer multiplication. :smile: